src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsServiceCollectionExtensions.cs

```diff
@@ -31,6 +31,7 @@ namespace Microsoft.Azure.WebJobs
     public static class WebJobsServiceCollectionExtensions
     {
         private const string SingletonConfigSectionName = "Singleton";
+        private const string CheckpointRetryConfigSectionName = "CheckpointRetryPolicy";
```

Comment:
We don't want extension specific logic/config in the core SDK. "CheckPoint" is only for EH, right?

Reply:
we also want to checkpoint CosmosDB and Kafka